packages/webpack-cli/lib/groups/StatsGroup.js

```diff
@@ -5,7 +5,7 @@ const { logger } = require('@webpack-cli/logger');
  */
 class StatsGroup extends GroupHelper {
     static validOptions() {
-        return ['minimal', 'none', 'normal', 'verbose', 'errors-warnings', 'errors-only'];
+        return ['minimal', 'none', 'normal', 'verbose', 'errors-warnings', 'errors-only', 'detailed'];
```

Comment:
@anshumanv Can you reorder values as here https://github.com/webpack/webpack/blob/master/schemas/WebpackOptions.json#L2605?